samples/boards/stm32/mco/boards/stm32f746g_disco.overlay

```diff
@@ -0,0 +1,44 @@
+/* The clock that is output must be enabled. */
+&clk_lse {
+	status = "okay";
+};
+
+&clk_hse {
+	status = "okay";
+};
+
+/* See reference manual:
+ *   01: LSE oscillator selected
+ */
+#define MCO1_SEL_LSE 1
+
+/* See referance manual:
+ *   100: division by 2
+ */
+#define MCO1_PRE_DIV_2 4
+
+&mco1 {
+	status = "okay";
+	clocks = <&rcc STM32_SRC_LSE MCO1_SEL(MCO1_SEL_LSE)>;
+	prescaler = <MCO1_PRE(MCO1_PRE_DIV_2)>;
+	pinctrl-0 = <&rcc_mco_1_pa8>; /* D10 (CN7) */
+	pinctrl-names = "default";
+};
+
+/* See reference manual:
+ *   10: HSE oscillator clock selected
+ */
+#define MCO2_SEL_HSE 2
+
+/* See referance manual:
```

Comment:
Typo:
```suggestion
/* See reference manual:
```

Nit: `RMxxxx` with appropriate RM number would be more appreciated (but it's not a big deal)